src/v2/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -1,85 +1,101 @@
 import React, { useState } from "react"
 import { RelayRefetchProp, createRefetchContainer, graphql } from "react-relay"
 import { get } from "v2/Utils/get"
+import styled from "styled-components"
 
 interface OrderRowProps {
   order: PurchaseHistory_me["orders"]["edges"][number]["node"]
+  hasDivider: boolean
 }
+
+const StyledImage = styled(Image)`
+  object-fit: cover;
+  height: 50px;
+  width: 50px;
+`
+
 const OrderRow = (props: OrderRowProps) => {
   const { order } = props
   const artwork = get(order, o => o.lineItems.edges[0].node.artwork)
 
+  const orderCreatedAt = DateTime.fromISO(order.createdAt).toLocaleString(
+    DateTime.DATE_SHORT
+  )
   if (!artwork) {
     return null
   }
 
+  const orderIsInactive =
+    order.state === "ABANDONED" || order.state === "CANCELED"
+
   return (
-    <Box py={1}>
-      <StackableBorderBox
+    <Box px="20px">
+      <Flex
+        py="15px"
         flexDirection="row"
         alignItems="center"
         justifyContent="space-between"
       >
-        <Flex width="350px">
-          <Flex height="auto" alignItems="center" mr={2}>
-            <Image
-              src={get(artwork, a => a.image.resized.url)}
-              alt={artwork.title}
-              width="55px"
-            />
-          </Flex>
-          <Flex flexDirection="column" justifyContent="center">
+        <Flex
+          alignItems="center"
+          justifyContent="center"
+          height="50px"
+          width="50px"
+          mr="15px"
+        >
+          <StyledImage
+            src={get(artwork, a => a.image.resized.url)}
+            alt={artwork.title}
+          />
+        </Flex>
+        <Flex flexDirection="column" justifyContent="center" width="100%">
+          {!orderIsInactive && (
             <Link
               href={`/orders/${order.internalID}/status`}
               underlineBehavior="hover"
             >
-              <Serif size="2" weight="semibold">
-                {artwork.artist_names}
-              </Serif>
+              <Sans size="3t">{artwork.artist_names}</Sans>
             </Link>
-            <Serif italic size="2" color="black60" lineHeight={1.3}>
-              {artwork.title}, {artwork.date}
-            </Serif>
-          </Flex>
+          )}
+          {orderIsInactive && <Sans size="3t">{artwork.artist_names}</Sans>}
+
+          <Sans size="3t" color="black60" lineHeight={1.3}>
+            {artwork.partner.name}
+          </Sans>
+          <Sans size="3t" color="black60" lineHeight={1.3}>
+            {orderCreatedAt}
+          </Sans>
         </Flex>
-        <Flex flexDirection="column" justifyContent="center">
+        <Flex flexDirection="column" alignItems="flex-end">
           <Sans
-            size="2"
+            size="3t"
             weight="medium"
             style={{ textTransform: "capitalize" }}
           >
-            {order.mode.toLowerCase()}
+            {order.itemsTotal ? order.itemsTotal : order.totalListPrice}
```

Comment:
Totally not something that needs to change, but this is a neat place for the nullish coalescing operator.

```suggestion
            {order.itemsTotal ?? order.totalListPrice}
```

Essentially if `order.itemsTotal` is null or undefined then `order.totalListPrice` will be used.

Reply:
Never seen that in my life but cool!